### PR TITLE
Anastasiya Apalkova - hw01

### PR DESCRIPTION
**Pull request by aapalkova**
For the first homework assignment I added a header, emphasis, two images, a link, and a list. Inserting a link, creating a header, and emphasis was easy, I followed [this guide](https://guides.github.com/features/mastering-markdown/).

I had some trouble inserting images because they would mess up the formatting of the text so I added a slide break to separate the images from the text using [this cheat sheet](https://rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf).  It also took me a lot of tries to get the sub bullets in my list to be indented because I was not indenting my nested list correctly. I figured this out when looking at the "blocked content in list items" section of [this document](https://pandoc.org/MANUAL.html#block-content-in-list-items). 

To figure out how to submit the assignment I watched the CFSS - homework workflow tutorial which explained everything very well. I also got a lot of practice pulling from GitHub, making changes, saving, committing, and pushing the commit to GitHub because I did it so many times when I made changes to my bullet list.

### CODE REVIEW

**Review by gilheany — COMMENTED** [review]
I like how you laid this out! The images and links all look great to me - I would maybe make the image a bit smaller, but it is a fun map to look at. I like your choices of what to emphasize, as well! I did almost the same. I hope you like MAPSS!

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
 # hw01
+
+
+# About Me
+My name is **Anastasiya Apalkova** and I am a student in the MAPSS program concentrating in psychology. 
+I was born in Ukraine and moved to Chicago with my family when I was seven years old. 
+***
```
Comment (majocamacho) [inline README.md:7]: Hi Anastasiya! It is great using *** to create a space between lines. In my assignment, I used double space and works well too!

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+![Ukraine Map](https://res.cloudinary.com/fen-learning/image/upload/c_limit,w_505,h_490/infopls_images/images/mukraine.gif) ![picture of me](pic4.jpg)
```
Comment (majocamacho) [inline README.md:8]: Using pictures give a better idea about who you are and the place you are from. I didn’t try it but I just learn with this review how to do it! Thank you for that! Maybe next time, you can try to make the map smaller for the images to have the same size.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+***
+I did my undergrad at [UW-Madison](https://www.wisc.edu/) where I majored in psychology and sociology.
```
Comment (majocamacho) [inline README.md:10]: I really liked that you used the link in UW Madison words, instead of just adding the link of the webpage.

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+I was born in Ukraine and moved to Chicago with my family when I was seven years old. 
+***
+![Ukraine Map](https://res.cloudinary.com/fen-learning/image/upload/c_limit,w_505,h_490/infopls_images/images/mukraine.gif) ![picture of me](pic4.jpg)
+***
+I did my undergrad at [UW-Madison](https://www.wisc.edu/) where I majored in psychology and sociology. 
+During my MAPSS year I will be completing a _thesis_ in social psychology as well as working on improving my 
+_computational and quantitative skills_ to prepare for PhD study. 
+During my free time I enjoy:
+
+* Reading
```
Comment (majocamacho) [inline README.md:15]: Nice use of the list as a way to organize and make the information easier to read!

File: README.md (hunk 2 above)
Comment (ssmarkow) [inline README.md:8]: I had a really hard time getting pictures to upload into Markdown. Nicely done.

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
...
+During my MAPSS year I will be completing a _thesis_ in social psychology as well as working on improving my 
+_computational and quantitative skills_ to prepare for PhD study. 
+During my free time I enjoy:
+
+* Reading 
+  + mostly non-fiction
+* Listening to podcasts 
+  + anything NPR
+  + true crime
+* And going on walks
```
Comment (ssmarkow) [inline README.md:20]: A good hobby in quarantine

File: README.md (hunk 2 above)
Comment (bensoltoff) [inline README.md:8]: Nice idea to store the image in the repo for embedding. Easy approach if the image is not already hosted online

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |    x          |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |  x         |              |            |
| **Achievement, creativity** |       x    |              |            |
| **Reproducibility**         |           |       x      |            |

**Pull request closed without merging**